ui/apps/platform/src/routePaths.ts

```diff
@@ -370,7 +371,7 @@ const routeRequirementsMap: Record<RouteKey, RouteRequirements> = {
     },
     'workload-cves': {
         featureFlagRequirements: allEnabled(['ROX_VULN_MGMT_WORKLOAD_CVES']),
-        resourceAccessRequirements: everyResource(['Deployment', 'Image']),
+        resourceAccessRequirements: everyResource(['Deployment', 'Image', 'Namespace']),
```

Comment:
@dvail I noticed that we can't see the entire Workload CVEs page if we don't have all of these. Were there any discussions, in the past, about displaying parts of Workload CVEs depending on permissions? I could see hiding just the "Namespace overview" button if we don't have "Namespace" permissions. Maybe that would be more appropriate for me to do in this case